packages/sdk/src/domain/model/HelioApiConnector.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  Currency,
+  FetchifyFindAddress,
+  FetchifyRetrieveAddress,
+  PaymentRequestType,
+  SwapRouteToken,
+  TokenQuoting,
+} from "@heliofi/common";
```

Comment:
is the linter configured correctly? I belive we have single quote rule instead of double quotes